__tests__/app.test.js

```diff
@@ -168,4 +168,44 @@ describe("/api/articles/:article_id/comments", () => {
         expect(response.body.msg).toBe("Bad request");
       });
   });
+  test("POST 201: inserts a new comment into the db and responds with the newly created comment", () => {
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send({
+        username: "icellusedkars",
+        body: "wow, this article is stupendous!",
+      })
+      .expect(201)
+      .then(({ body }) => {
+        const { new_comment } = body;
+        expect(new_comment.author).toBe("icellusedkars");
+        expect(new_comment.body).toBe("wow, this article is stupendous!");
+        expect(new_comment.article_id).toBe(1);
+        expect(new_comment.votes).toBe(0);
+        expect(typeof new_comment.created_at).toBe("string");
+      });
+  });
+  test("POST 400: responds with an error when given an invalid username in post request body", () => {
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send({
+        username: "tiahontoast",
+        body: "I'm not a user of this site and I hate this article!",
+      })
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe("Bad request");
+      });
+  });
+  test("POST 400: responds with an error when given a post request body with no body property", () => {
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send({
+        username: "icellusedkars",
+      })
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe("Bad request");
+      });
+  });
```

Comment:
Solid tests so far 💪  though could you also add a test for a 404 with a valid but non-existent article_id
Like `/api/articles/9999/comments`
You could also add a test for if the `username` property is missing